src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,77 @@
+.menu {
+  background-color: $mainColor;
+
+  &__content {
+    height: 100vh;
+    box-sizing: border-box;
```

Comment:
It is a good tone to set this property for all elements(*) and don't change without extra need